content/articles/a-beginners-guide-on-how-to-become-a-blockchain-developer/index.md

```diff
@@ -0,0 +1,181 @@
+
+Taking a look at the 1-year chart on [DeFi Pulse](https://defipulse.com/), you will notice how the adoption of DeFi has skyrocketed over the last few months.
+
+Currently, most of the DeFi runs on the Ethereum blockchain. Popular DeFi project in the market today include [Maker](https://makerdao.com/en/), [Aave](https://aave.com/), [Compound](https://compound.finance/), [Polygon](https://polygon.technology/) and [Uniswap](https://uniswap.org/). Maker, Aave, and Compound are used as lending platforms, Polygon as a payment platform, and Uniswap for decentralized exchange (DEX).
+
+As a blockchain developer, you can build platforms such as these.
+
+2. Supply Chain
+
+The supply chain system involves tracking the movement of goods that end up turning into end products. Big companies such as Walmart use it to increase supply chain transparency.
+
+Using blockchain technology helps ensure that distributors across the supply chain are honest and transparent.
+
+3. Digital Identity
+
+Cardano has partnered with [Atala Prism](https://atalaprism.io/app) to enable the issuance of digital identities and credentials that users own, control, and share easily, both securely, and privately. This will be built on the Cardano blockchain. Building it on Cardano's blockchain eliminates the need for having a third party offer verification services. They recently secured a deal with the Ethiopian government where they are planning to issue over 5 million students with digital identities. Learn more about the deal [here](https://www.forbes.com/sites/rachelwolfson/2019/04/30/cardano-founder-launches-enterprise-blockchain-framework-in-collaboration-with-ethiopian-government/?sh=50d7424e4e10). 
+
+As a blockchain developer, you can build digital identity systems that secure digital IDs, degrees, certificates, transcripts, examinations, and badges, etc. on the blockchain.
+
+4. Healthcare
+
+In healthcare, blockchain can be used for record-keeping & medical history tracking, patient & drug identity validation, and to offer direct healthcare service payments. This [article](https://www.section.io/engineering-education/blockchain-as-a-revolution-in-healthcare/) explains clearly how blockchain technology is used to enable these services in healthcare. 
+
+
+These are a few use cases for blockchain technology. It can also be used to secure title deeds in real estate and to secure voting systems from tamper in democratic countries.
+
+### Wrapping Up
+
+In this article, I have explained how you can become a blockchain developer and take advantage of this incredibly scarce and lucrative field. It is wrong to assume that just because you have no prior coding experience, that this is not for you. Blockchain development is a skill you master over time. Just follow this guide and continuously try and practice. With blockchain development, practice makes perfect.
```

Comment:
```suggestion
In this article, I have explained how you can become a blockchain developer and take advantage of this lucrative field. It is wrong to assume that since you may have no prior coding experience, this niche is not for you. As with any skill, blockchain development is mastered over time. This guide gives a starting point. However, it is your responsibility to continuously learn, try and practice. As they say, practice makes perfect.
```

Reply:
Consider adding references to add value for the reader